guides/common/modules/con_kickstart-workflow.adoc

```diff
@@ -14,4 +14,4 @@ When you run a {ProjectName} Kickstart script, the script performs the following
 . It runs user defined snippets.
 
 .Additional resources
-For more information about Kickstart, see {RHELDocsBaseURL}8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in _Performing an advanced RHEL 8 installation_.
+For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL_.
```

Comment:
```suggestion
For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_.
```
I think it's better to include the RHEL version in the link description.
Unlike above, the title already includes the RHEL abbreviation, so let's just add the version.
Normally, we would use the exact wording of the title, but I'd prefer to include the version number for clarity and context.

Please, also change the other occurences.